src/routes/get/+page.svelte

```diff
@@ -0,0 +1,49 @@
+<script>
+	import Navbar from '../../components/interface/navbar.svelte';
+</script>
+
+<Navbar />
+<div class="section">
+	<div class="width">
+		<div id='load' class="section-header get">
+			<h1>
+				Prépare toi à découvrir une révolution !
+			</h1>
+			<p>
+				Nous préparons ton arrivé(e), tu vas bientôt pouvoir télécharger Papillon.
```

Comment:
```suggestion
				Nous préparons ton arrivée, tu vas bientôt pouvoir télécharger Papillon.
```

"arrivée" toujours au féminin c'est un nom